backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/FileWrapper.java

```diff
@@ -196,11 +196,17 @@ public String readString (String charset) {
 	/** Reads the entire file into a byte array.
 	 * @throw GdxRuntimeException if the file handle represents a directory, doesn't exist, or could not be read. */
 	public byte[] readBytes () {
+		return readBytes(read());
+	}
+
+	/** Reads the entire file into a byte array.
+	 * @param input inputstream to use. The inputstream is closed after use.
+	 * @throw GdxRuntimeException if the file handle represents a directory, doesn't exist, or could not be read. */
+	public byte[] readBytes (InputStream input) {

```

Comment:
Could maybe use StreamUtils `copyStreamToByteArray`?
https://github.com/libgdx/libgdx/blob/master/gdx/src/com/badlogic/gdx/utils/StreamUtils.java#L83

Reply:
Ah yes, that would be an even better way.